src/App.tsx

```diff
@@ -9,34 +11,34 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">
-        Selected tab is Tab 1
-      </h1>
+export class App extends React.Component {
```

Comment:
this task is in the topic after the `hooks`, and the initial entry of the component was functional, then the functional approach is used in this and the following tasks